### PR TITLE
Removed unneccessary (redudant) code from code/utils/linear_modeling.py

### PR DESCRIPTION
**Pull request by AlonDaks**

### CODE REVIEW

**Comment by lisaannyu** [comment]
I think you can remove everything from get_train_day to the end.  It's in the rf.py module or in the get_rf_design_matrix function

**Comment by lisaannyu** [comment]
looks good

File: code/utils/linear_modeling.py [hunk 1]
```diff
@@ -18,16 +18,14 @@ def get_design_matrix():
...
     n_trs = data.shape[-1]
-    X = np.ones((n_trs, 4))
+    X = np.ones((n_trs, 3))
     ss = ssm.SceneSlicer('test_data.nii','scenes.csv') 
     day_night, int_ext = ss.get_scene_slices()
-    X[:, 0] = day_night
-    X[:, 1] = int_ext
-    # X[:, 2] = pos
+    X[:, 1] = day_night
     X[:, 2] = np.linspace(-1, 1, n_trs)
```
Comment (lisaannyu) [inline code/utils/linear_modeling.py]: This isn't super important, but I would prefer to have the linear drift column next to the column of ones, just so all the columns that we are less interested in (i.e. intercept and linear drift) are next to each other

**Comment by AlonDaks** [comment]
Ok, I will make that fix in my next PR

File: code/utils/linear_modeling.py (hunk 1 above)
Comment (AlonDaks) [inline code/utils/linear_modeling.py]: Ok, I will make that fix in my next PR

**Pull request merged**